src/kinematics_plugin.cpp

```diff
@@ -127,29 +120,33 @@ struct BioIKKinematicsPlugin : kinematics::KinematicsBase {
 
   BioIKKinematicsPlugin() { enable_profiler = false; }
 
-  virtual const std::vector<std::string> &getJointNames() const {
+  virtual const std::vector<std::string> &getJointNames() const override {
     LOG_FNC();
     return joint_names;
   }
 
-  virtual const std::vector<std::string> &getLinkNames() const {
+  virtual const std::vector<std::string> &getLinkNames() const override {
     LOG_FNC();
     return link_names;
   }
 
-  virtual bool getPositionFK(const std::vector<std::string> &link_names,
-                             const std::vector<double> &joint_angles,
-                             std::vector<geometry_msgs::Pose> &poses) const {
+  virtual bool getPositionFK(
+    [[maybe_unused]] const std::vector<std::string> &,
+    [[maybe_unused]] const std::vector<double> &,
+    [[maybe_unused]] std::vector<geometry_msgs::msg::Pose> &) const override {
```

Comment:
These parameters are already unnamed so you don't need `[[maybe_unused]]`.

Reply:
Addressed in [e9c01ee](https://github.com/PickNikRobotics/bio_ik/pull/1/commits/e9c01ee4ef00ced5d79bb788ab01a067e93e922c)